glean-core/src/system.rs

```diff
@@ -79,3 +81,31 @@ pub const OS: &str = "Solaris";
     target_os = "solaris",
 )))]
 pub const OS: &str = "unknown";
+
+#[cfg(any(target_os = "macos", target_os = "windows"))]
+/// Returns Darwin kernel version for MacOS, or NT Kernel version for Windows
+pub fn get_os_version() -> String {
+    sys_info::os_release().unwrap()
```

Comment:
This could panic if it can't read the version. We should probably turn this into "unknown" in case of errors.

```suggestion
    sys_info::os_release().unwrap_or_else(|| "unknown".to_string())
```